scripts/gen_vimdoc.py

```diff
@@ -1096,8 +1115,9 @@ def main(config, args):
             try:
                 title, helptag, section_doc = sections.pop(filename)
             except KeyError:
-                msg(f'warning: empty docs, skipping (target={target}): {filename}')
-                msg(f'    existing docs: {sections.keys()}')
+                msgs.append(
+                        f'warning: empty docs, skipping (target={target}): {filename}')
+                msgs.append(f'    existing docs: {sections.keys()}')
```

Comment:
what did we gain here?

Reply:
No global variables, which in hindsight wasn't a big win :)

Reply:
Wait, no, this one was because adding a function `msg` for an elementary operation `append` seemed more confusing than useful to me personally.

Reply:
You're right that needless indirection is pure cost. But this is intended to be an abstraction for "message collection". The internals of `msg()` may change later. So, the purpose is not to avoid `append()`, but to have an interface for message collection.